### DISCUSSION TITLE
Content Search

### DISCUSSION THREAD

**Discussion by nicpitsch** [2sic/2sxc#3486] [Q&A] [answered]
How can [Oqtane Content Search](https://github.com/oqtane/oqtane.framework/discussions/4441) be implemented by 2sxc App's / Content?

**Comment by iJungleboy** [comment, marked as answer]
Not yet.

Oqtanes Search Indexer is still very new, so 2sxc doesn't communicate with it yet. 

But I do assume that the entire setup will be similar to dnn, so I think it should be doable in 1-2 days of work. 

But someone's got to do it ;)
